### DISCUSSION TITLE
Custom webhook url has always the same port

### DISCUSSION THREAD

**Discussion by korontanis** [carlosedp/cluster-monitoring#141] [Q&A] [answered]
Hi,

I am declaring a custom webhook url at the alertmanager-config.yaml and no matter what port I am declaring it always changes back to 3000. Do you have any what causes this ?

kind: ConfigMap
apiVersion: v1
metadata:
  name: alertmanager-config
  namespace: monitoring
data:
  config.yml: |-
    global:
    route:
      receiver: custom_webhook
      group_by: ['alertname', 'priority']
      group_wait: 10s
      repeat_interval: 1m
      routes:
        - receiver: custom_webhook
          match:
            severity: deployed
          group_wait: 10s
          repeat_interval: 1m
    receivers:
    - name: custom_webhook
      webhook_configs:
      - url: 'http://0.0.0.0:4000/webhook'

![image](https://user-images.githubusercontent.com/56536805/149370920-c4a0f56b-d562-42e1-9bc2-88719825edf2.png)

**Comment by carlosedp** [comment, marked as answer]
This yaml is dynamically generated on `make` so your changes will be overridden. This config is generated by the ksonnet libs in prometheus-operator so you might need to override in your own config.
